homeassistant/components/apcupsd/sensor.py

```diff
@@ -34,392 +34,343 @@
         key="alarmdel",
         translation_key="alarm_delay",
-        name="UPS Alarm Delay",
     ),
     "ambtemp": SensorEntityDescription(
         key="ambtemp",
-        name="UPS Ambient Temperature",
+        translation_key="ambient_temperature",
         native_unit_of_measurement=UnitOfTemperature.CELSIUS,
         device_class=SensorDeviceClass.TEMPERATURE,
         state_class=SensorStateClass.MEASUREMENT,
     ),
     "apc": SensorEntityDescription(
         key="apc",
         translation_key="apc_status",
-        name="UPS Status Data",
         entity_registry_enabled_default=False,
     ),
     "apcmodel": SensorEntityDescription(
         key="apcmodel",
         translation_key="apc_model",
-        name="UPS Model",
         entity_registry_enabled_default=False,
     ),
     "badbatts": SensorEntityDescription(
         key="badbatts",
         translation_key="bad_batteries",
-        name="UPS Bad Batteries",
     ),
     "battdate": SensorEntityDescription(
         key="battdate",
         translation_key="battery_replacement_date",
-        name="UPS Battery Replaced",
     ),
     "battstat": SensorEntityDescription(
         key="battstat",
         translation_key="battery_status",
-        name="UPS Battery Status",
     ),
     "battv": SensorEntityDescription(
         key="battv",
-        name="UPS Battery Voltage",
+        translation_key="battery_voltage",
         native_unit_of_measurement=UnitOfElectricPotential.VOLT,
         device_class=SensorDeviceClass.VOLTAGE,
         state_class=SensorStateClass.MEASUREMENT,
     ),
     "bcharge": SensorEntityDescription(
         key="bcharge",
-        name="UPS Battery",
+        translation_key="battery",
         native_unit_of_measurement=PERCENTAGE,
         device_class=SensorDeviceClass.BATTERY,
         state_class=SensorStateClass.MEASUREMENT,
     ),
     "cable": SensorEntityDescription(
         key="cable",
         translation_key="cable_type",
-        name="UPS Cable Type",
         entity_registry_enabled_default=False,
     ),
     "cumonbatt": SensorEntityDescription(
         key="cumonbatt",
         translation_key="total_time_on_battery",
-        name="UPS Total Time on Battery",
         state_class=SensorStateClass.TOTAL_INCREASING,
     ),
     "date": SensorEntityDescription(
         key="date",
         translation_key="date",
-        name="UPS Status Date",
         entity_registry_enabled_default=False,
     ),
     "dipsw": SensorEntityDescription(
         key="dipsw",
         translation_key="dip_switch_settings",
-        name="UPS Dip Switch Settings",
     ),
     "dlowbatt": SensorEntityDescription(
         key="dlowbatt",
         translation_key="low_battery_signal",
-        name="UPS Low Battery Signal",
     ),
     "driver": SensorEntityDescription(
         key="driver",
         translation_key="driver",
-        name="UPS Driver",
         entity_registry_enabled_default=False,
     ),
```

Comment:
If this is a version, do we already expose it in the device info? (sw version or fw version)?